tests/performance/load/groups/alerts.js

```diff
@@ -0,0 +1,40 @@
+import {Client, StatusOK} from 'k6/net/grpc';
+import {check, group} from 'k6';
+
+const GRPC_PORT = ":443";
+
+const client = new Client();
+
+client.load(['proto/stackrox', 'proto/googleapis'], 'api/v1/alert_service.proto');
```

Comment:
nitpick: Is there a reason why we are using symlinks here instead of relative imports? I would avoid symlinks if possible. It's just a personal preference.

Reply:
It was requested here
https://github.com/stackrox/stackrox/pull/14265#discussion_r1954820985